app/coffeescripts/gradebook2/Gradebook.coffee

```diff
@@ -330,7 +330,7 @@ define [
     addRow: (student) =>
       student.computed_current_score ||= 0
       student.computed_final_score ||= 0
-      student.secondary_identifier = student.sis_login_id || student.login_id
+      student.secondary_identifier = student.email || sis_login_id || student.login_id
```

Comment:
i think there is a bug here where it should be '''student. sis_login_id''' in the second conditional